base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/commands/DeleteConfigsHandler.java

```diff
@@ -48,11 +54,49 @@ public DeleteConfigsHandler() {
 	public void safeExecute(Shell shell) {
```

Comment:
I may be misunderstanding this function. Are we meant to be actually deleting the configs or simply removing them from the recently loaded list?

Reply:
Function description is vague, it opens the delete configs and also deletes the config with `configService.uncheckedWrite((dialog.selectedConfigs()))` it then removes deleted configs from recently used list using `Configurations.getInstance().removeNameFromRecentlyLoadedConfigList(item)` if no error is detected.

Reply:
Function description is vague, it opens the delete configs and also deletes the config with configService.uncheckedWrite((dialog.selectedConfigs())) it then removes deleted configs from recently used list using Configurations.getInstance().removeNameFromRecentlyLoadedConfigList(item) if no error is detected.

Reply:
This makes more sense thank you